### ISSUE TITLE
Keyboard shortcuts: No vocalization

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#2032]
## Bug Report

**Problematic behavior**
When using keyboard shortcuts to format following the shortcuts recommendations, if I switch from a numbered list to a toggle list, the state change is not vocalized.

**Expected behavior/code**
If I activate a keyboard shortcut, its action should be announced.

**Steps to Reproduce**
1. Create a new Doc
2. Enable NVDA or VO speech synthesis
3. Enter text and use keyboard shortcuts
4. And then the bug happens!

**Additional context/Screenshots**
[Video ](https://github.com/user-attachments/assets/1b66bcb7-348f-469e-8297-9d1e00c526c2
)

**Critères RGAA**
Critère 7.1 : [Chaque script est-il, si nécessaire, compatible avec les technologies d'assistance ?](https://accessibilite.numerique.gouv.fr/methode/criteres-et-tests/#7.1)

**Impact** 
The user does not know whether their shortcut has been taken into account or not in the formatting of their content. The shortcuts are only documented if you are in the list of proposed actions.
